fe/topology.py

```diff
@@ -578,12 +578,22 @@ def parameterize_nonbonded(self, ff_q_params, ff_lj_params):
 
         # both mol_a and mol_b are standardized.
         _, nb_potential = super().parameterize_nonbonded(ff_q_params, ff_lj_params)
-        mol_c = Chem.CombineMols(self.mol_a, self.mol_b)
-        qlj_params = standard_qlj_typer(mol_c)
 
-        # ligand is already decharged, and super-ligand state should have half the epsilons
-        src_qlj_params = jax.ops.index_update(qlj_params, jax.ops.index[:, 2], qlj_params[:, 2] * 0.5)
-        dst_qlj_params = qlj_params
+        qlj_params_a = standard_qlj_typer(self.mol_a)
+
+        src_qlj_params_a = jax.ops.index_update(qlj_params_a, jax.ops.index[:, 0], qlj_params_a[:, 0] * 0.5)
```

Comment:
Would be handy to have a comment explaining the goal of having the host only feel a single ligand in the super state.